library/direct_message_check.inc

```diff
@@ -430,6 +431,9 @@ function phimail_close($fp)
 
 function phimail_logit($success, $text, $pid = 0, $event = "direct-message-check")
 {
+    if (!$success) {
+        (new \OpenEMR\Common\Logging\SystemLogger())->errorLogCaller($event, ['success' => $success, 'text' => $text, 'pid' => $pid]);
```

Comment:
place in use block above so can use shortened form

Reply:
Fixed